### ISSUE TITLE
Regression due to "gigasamples" update

### ISSUE THREAD

**Issue by kode54** [losnoco/Cog#150]
```
The last "gigaseconds" update brought some regressions with it:

— when remaining time is positive, it is not centered inside its window;
— when remaining time is negative, it is again preceded by a hyphen instead of a minus and for longer tracks the whole digit is cut off.
```

**Comment by kode54** [comment]
Can't fix the whining about how it's not a unicode minus sign, this new text field doesn't support UTF-8 encoded text.

**Comment by JanX2** [comment]
@kode54 
That’s interesting. I couldn’t figure out why a hyphen was used. Typographically (former visual designer) this don’t make any sense to me. 

I am aware of the centering issue, but couldn’t find out how to resolve it complete, just make improvements. See screenshots in #148.

Can you make a screenshot of the cut off digits issue? I have not seen this on 10.15.7. It may be limited to Big Sur…

**Comment by kode54** [comment]
The minus sign is something a user who personally emails me keeps on about. I have no idea what their issue is, unless they're using a screen reader.

Meanwhile, on their 10.13.x system, the box does this:
![Screen Shot 2021-05-08 at 21 13 14](https://user-images.githubusercontent.com/796316/117552612-5efc9700-b001-11eb-9eb0-52a47cd44f0c.png)

Yes, that's right, the box doesn't dynamically resize itself for them. Or their window is so cramped, it's forced to a maximum size that's equivalent to the minimum size.

Another issue they're experiencing, too:
The inspector window doesn't remember its position. I wonder if I should tell them that it was a new feature to make the inspector window always open right next to the main window?

**Comment by nevack** [comment]
At this point I think it's better to revert this change and review it properly in more granular PRs

**Comment by kode54** [comment]
It works fine on 11.0. And apparently the 10.15 that its developer is using. I guess we can revert it anyway.

**Comment by kode54** [comment]
@JanX2 Another regression I just noticed: When playing a looping track beyond its end time, the "remaining" time doesn't clip off at -0:00, but instead continues forward into the positive time, resulting in a timestamp that's the total play position minus the duration of the file.

**Comment by JanX2** [comment]
@kode54 The looping issue is an interesting one. 😂

**Comment by JanX2** [comment]
@kode54 
After thinking about this for a bit, my suggestion is to work around the clipped current position text by setting the minimum size of the item to a fixed high value on 10.13.

**Comment by kode54** [comment]
Check this out now. The field is sort of centered. The only non-centering is due to the fact that it tries to pad the text based on how many digits the track duration requires, plus the width of the numeric minus sign for time remaining mode. It also uses monospaced numbers in the system font. The end result is a field that has a fixed width for the duration of the current track.
